zstacklib/zstacklib/utils/lvm.py

```
@@ -2845,9 +2809,33 @@ def report_config_changed():
+REPAIR_VG_CMDS = {"vgchange"}
 def subcmd(cmd, timeout=lvm_cmd_timeout_with_locking):
-    if cmd in ["lvs", "pvs", "vgs"]:
-        return "%s --nolocking -t" % cmd
-    elif cmd in ["lvchange", "lvcreate", "lvrename", "lvresize", "lvextend", "lvremove"]:
-        return "timeout -s SIGKILL %s %s"% (timeout, cmd)
-    return cmd
\ No newline at end of file
+    argv = [cmd]
+
+    if cmd in NOLOCK_CMDS:
+        return argv + ["--nolocking", "-t"]
```

Ruff (0.15.2)

[warning] Consider `[*argv, "--nolocking", "-t"]` instead of concatenation

Replace with `[*argv, "--nolocking", "-t"]`

(RUF005)